### PR TITLE
[makefile] Update EFSW dep to premake5

### PR DESCRIPTION
**Pull request by SteveRussell33**
Fix for https://github.com/VCVRack/VCV-Prototype/issues/54#issuecomment-922844278

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,14 +29,14 @@ endif
 
 
 # Entropia File System Watcher
-efsw := dep/lib/libefsw-static-release.a
+efsw := dep/lib/efsw-static-release.lib
```
Comment (JerrySievert) [inline Makefile:32]: you might want to make this conditional on OS

File: Makefile [hunk 2]
```diff
@@ -29,14 +29,14 @@ endif
...
 DEPS += $(efsw)
 OBJECTS += $(efsw)
 $(efsw):
-	cd efsw && premake4 gmake
-	cd efsw && $(MAKE) -C make/* config=release efsw-static-lib
+	cd efsw && premake5 gmake
```
Comment (JerrySievert) [inline Makefile:36]: this completely breaks building on macOS, should be conditional

File: Makefile [hunk 3]
```diff
@@ -29,14 +29,14 @@ endif
...
+	cd efsw && $(MAKE) -C make/* config=release_x86_64 efsw-static-lib
 	mkdir -p dep/lib dep/include
-	cd efsw && cp lib/libefsw-static-release.a $(DEP_PATH)/lib/
+	cd efsw && cp lib/efsw-static-release.lib $(DEP_PATH)/lib/
```
Comment (JerrySievert) [inline Makefile:39]: see conditional notes

**Comment by SteveRussell33** [comment]
Sorry, trying to fix windows issues in https://github.com/VCVRack/VCV-Prototype/issues/54
Will redo this.

**Pull request closed without merging**